dnf/cli/commands/makecache.py

```diff
@@ -33,7 +33,7 @@
 
 
 class MakeCacheCommand(commands.Command):
-    aliases = ('makecache',)
+    aliases = ('makecache', 'mc')
```

Comment:
It may also make sense to alias this to `refresh` / `ref`, as it does the same thing `Zypper refresh` does.